index.html

```diff
@@ -73,6 +76,237 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
+            <a href="#"
+              ><img class="social_icon" src="./images/github.svg" alt="Github"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/Vector (1).svg" alt="Diot"
+            /></a>
+          </li>
+        </ul>
+        <div class="action">
+          <button type="button" class="default_btn">Get my resume</button>
+        </div>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Languages
+            <img
+              class="down_arrow_icon"
+              src="./images/Icon - Chevron - Down - Larger.png"
+              alt="down"
+            />
+          </h3>
+          <li class="ul_item">
+            <img src="./images/Ellipse 1.png" alt="Javascript" /><span
+              >Javascript</span
+            >
+          </li>
+          <li class="ul_item">
+            <img src="./images/Ellipse 1 (1).png" alt="HTML" /><span>HTML</span>
+          </li>
+          <li class="ul_item">
+            <img src="./images/Ellipse 1 (2).png" alt="CSS" /><span>CSS</span>
+          </li>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Frameworks
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Skills
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+      </section>
+      <section class="contact_form">
+        <h2 class="title">Contact me</h2>
+        <p class="contact_form_des">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form
+          class="form_container"
+          action="https://formspree.io/f/mnqroaaw"
+          method="POST"
+        >
+          <fieldset class="form_inputs">
+            <input
+              class="form_input"
+              name="name"
+              type="text"
+              placeholder="Your Name"
+              maxlength="30"
+              required
+            />
+            <input
+              class="form_input"
+              name="email"
+              type="email"
+              placeholder="example@me.com"
+              required
+            />
+            <textarea
+              class="form_text_area"
+              name="message"
+              placeholder="Write your message here"
+              maxlength="500"
+              required
+            ></textarea>
+            <button type="submit" class="btn_submit">Get It Touch</button>
+          </fieldset>
```

Comment:
- [ ] Kindly make sure to match your design with [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and fix the alignment of text and fix also the background to match the one in Figma.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/200538242-70c36e8d-2785-4577-b1ae-dcebe088f6c6.png) | ![image](https://user-images.githubusercontent.com/53402600/200538306-d6159a99-827d-4e7d-a175-4e22db5c8f88.png) |.